web/portal/client/cypress/e2e/Calendar/HolidayDate/HolidayDate.cy.js

```diff
@@ -0,0 +1,168 @@
+  });
+
+  ///////////////////////
+  // POST
+  ///////////////////////
+  it('add HolidayDate', () => {
+    cy.usePactIntercept(
+      {
+        method: 'POST',
+        url: '**/api/client/holiday_dates*',
+        response: NewHolidayDate.response,
+        matchingRules: NewHolidayDate.matchingRules,
+      },
+      'createHolidayDate'
+    );
+
+    cy.get('[aria-label=Add]').click();
+
+    const { name, eventDate } = NewHolidayDate.request;
+    cy.fillTheForm({
+      name,
+      eventDate,
+    });
+
+    cy.get('header').should('contain', 'Holiday dates');
+    cy.usePactWait('createHolidayDate')
+      .its('response.statusCode')
+      .should('eq', 201);
+  });
+
+  ///////////////////////////////
+  // PUT
+  ///////////////////////////////
+  it('edit HolidayDate', () => {
+    cy.intercept('GET', '**/api/client/holiday_dates/1', {
+      ...HolidayDateItem,
+    }).as('getHolidayDate-1');
+
+    cy.usePactIntercept(
+      {
+        method: 'PUT',
+        url: `**/api/client/holiday_dates/${EditHolidayDate.response.body.id}`,
+        response: EditHolidayDate.response,
+      },
+      'EditHolidayDate'
+    );
+
+    cy.get('svg[data-testid="EditIcon"]').first().click();
+    const { name, eventDate, locution } = EditHolidayDate.request;
+    cy.fillTheForm({
+      name,
+      eventDate,
+      locution,
+    });
+    cy.contains('Holiday dates');
+    cy.usePactWait(['EditHolidayDate'])
+      .its('response.statusCode')
+      .should('eq', 200);
+  });
+
+  ///////////////////////
+  // DELETE
+  ///////////////////////
+  it('delete HolidayDate', () => {
+    cy.intercept('DELETE', '**/api/client/holiday_dates/*', {
+      statusCode: 204,
+    }).as('deleteHolidayDate');
+
+    cy.get('td button > svg[data-testid="DeleteIcon"]').first().click();
+    cy.contains('Remove element');
+    cy.get('div.MuiDialog-container button')
+      .filter(':visible')
+      .contains('Yes, delete it')
+      .click();
+
+    cy.get('header').should('contain', 'Holiday dates');
+
+    cy.usePactWait(['deleteHolidayDate'])
+      .its('response.statusCode')
+      .should('eq', 204);
+  });
+
+  ///////////////////////
+  // POST
+  ///////////////////////
+  it('add HolidayDateRange', () => {
+    cy.usePactIntercept(
+      {
+        method: 'POST',
+        url: '**/api/client/holiday_dates_range*',
+        response: NewHolidayDateRange.response,
+        matchingRules: NewHolidayDateRange.matchingRules,
+      },
+      'createHolidayDateRange'
+    );
+
+    cy.get('button svg[data-testid="MoreHorizIcon"]').first().click();
+
+    cy.contains('Add Holiday date range').click();
+
+    const { startDate, endDate, locution, name } = NewHolidayDateRange.request;
+    cy.fillTheForm({
+      startDate,
+      endDate,
+      locution,
+      name,
+    });
+
+    cy.get('header').should('contain', 'Holiday dates');
+    cy.usePactWait('createHolidayDateRange')
+      .its('response.statusCode')
+      .should('eq', 201);
+  });
```

Comment:
HolidayDateRange is a different entity, so it should be in separate test files with its own fixtures.